### ISSUE TITLE
How do I start fullsizable in fullscreen mode by default?

### ISSUE THREAD

**Issue by ProTechnologist** [MSchmidt/jquery-fullsizable#16]
Hi,

Thanks for writing a great plugin and I'm using in one of my own upcoming open source project that uses Electron/Chromium.

The question is:
How can I start showing the pictures in fullscreen mode ?

I looked into your code and have found an internal function toggleFullscreen which I can't call on page load event outside he plugin as the plugin instance isn't returning from the constructor.

I just want to skip a step that users won't have to click on thumbnails and then click on the fullscreen button.

**Comment by Hudik** [comment]
You can not request fullscreen mode on pageload event because of "The user interaction task source" condition. (see Model in https://fullscreen.spec.whatwg.org/ , This task source is used for features that react to user interaction, for example keyboard or mouse input.)
 If you are interested in going fullscreen in 1 click, try https://github.com/blueimp/Gallery#lightbox-setup.
There is more flexible and extendable product.

**Comment by ProTechnologist** [comment]
Thanks for quick response.

I have checked the blueimp and it is little bit overkill for what I need as it is making my electron app a bit sluggish whereas fullsizable run super-smooth even in my test VM environment with low specs.

Right now, I have put this requirement to low priority and going to publish the project as-is on github in new few hours.

Once this is done, I might look into the possibility of simulating user gesture (something what blueimp is doing at the moment) within the fullsizable.
